### DISCUSSION TITLE
[#1166398] Selection resizing typo in Quick Start

### DISCUSSION THREAD

**Discussion by ForestOfLight** [Mojang/minecraft-editor#158] [Bugs]
In the Quick Start menu, it clearly reads: "Resize using corner gizmos, or hold **Ctrl** (increase) or **Shift** (decrease) with arrow keys and page up/page down." Ctrl and Shift are mislabeled and should be swapped. It should read: "Resize using corner gizmos, or hold **Shift** (increase) or **Ctrl** (decrease) with arrow keys and page up/page down."

**Comment by minecraftchloe** [comment]
Going to be doing a copy overhaul in the next few weeks on Quick Start!
